conf/broker.conf

```
@@ -1163,6 +1163,9 @@ loadBalancerEnabled=true
 # Percentage of change to trigger load report update
 loadBalancerReportUpdateThresholdPercentage=10
 
+# minimum interval to update load report
+loadBalancerReportUpdateMinIntervalMilliSeconds=5000
```

```suggestion
loadBalancerReportUpdateMinIntervalMillis=5000
```

Just keep consistent with other configuration names in the broker.conf

---

updated.